lib/machinery_helper.rb

```diff
@@ -34,17 +31,25 @@ def initialize(s)
     @system = s
     @arch = @system.arch
 
-    @local_helpers_path = "/usr/share/machinery/helpers"
+    @local_helpers_path = File.join(Machinery::ROOT, "machinery-helper")
+    @remote_helpers_path = File.join(Machinery::REMOTE_HELPERS_PATH, "machinery-helper")
```

Comment:
It should be `@remote_helper_path` (singular), as there always is only one helper installed on the remote machine.